about/diversity.md

```diff
@@ -22,12 +22,12 @@ This is an ongoing process. We are talking to our program chairs, program commit
 
 Here are some ways you can help us build a more diverse conference experience:
 
-* Recommend appropriate speakers and/or program committee members to the conference chairs. You may also send an email to pytn-staff.
+* Recommend appropriate speakers and/or program committee members to the conference chairs. You may also send an email to [organizers@pytennessee.org](mailto:{{site.data.event.organizers_email}}).
```

Comment:
Should we use the variable for the display text as well as the mailto link?

Reply:
I can try again but I was having issues with it rendering for some reason. It was like it didn't like the variable inside their

Reply:
Huh, gave it the old college try and it let me do it this time. Not sure why it hated how I did it before